tests/test_transform.py

```diff
@@ -31,3 +34,29 @@ def test_transform_xml_rules_to_yaml():
         )
     except ValidationError as e:
         assert True is False, 'The validation of the generated YAML rule failed. ' + e.message
+
+
+# Polarion TC 376
+def test_transform_code_with_openrewrite():
```

Comment:
We might have to automate this test for multiple applications so it would be best not to use the app name in the test .

Reply:
`openrewrite` is not the app name but the subcommand used to transform the code.

![image](https://github.com/konveyor-ecosystem/kantra-cli-tests/assets/117646518/d07d1248-ec84-4cfd-9430-cbc1791c48fe)

Reply:
Yeah I put the comment at the wrong place . I meant , remove "complete_duke" app name. 

Looks perfect now :+1: